client/post-editor/editor-notice/index.jsx

```diff
@@ -89,6 +89,31 @@ export class EditorNotice extends Component {
 					comment: 'Editor: Message displayed when a post is published, with a link to the site it was published on.'
 				} );
 
+			case 'scheduled':
```

Comment:
>Though translations could be a challenge.

Yeah, I'd considered this CPT-ified as of #6794. The only label we could dynamically generate from the post type is `view`, already included here:

https://github.com/Automattic/wp-calypso/blob/7141b4e/client/post-editor/editor-notice/index.jsx#L144-L151